apps/demos/Demos/Chat/Customization/Angular/app/app.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="demo-container">
+  <div class="chat-container">
+    <dx-chat
+      #chat
+      [user]="currentUser"
```

Comment:
Please set `[height]="710"`
It's not okay that height jumps when u disable day headers or send new messages.
Do it in all frameworks.

Reply:
[37c8016](https://github.com/DevExpress/DevExtreme/pull/28482/commits/37c8016387efd0b5991f5f9cb043fba1c686bd51)